tests/pipelines/audioldm2/test_audioldm2.py

```diff
@@ -361,7 +380,18 @@ def test_audioldm2_negative_prompt(self):
 
         audio_slice = audio[:10]
         expected_slice = np.array(
-            [0.0025, 0.0018, 0.0018, -0.0023, -0.0026, -0.0020, -0.0026, -0.0021, -0.0027, -0.0020]
+            [
+                0.00257826,
+                0.00173665,
+                0.00182106,
+                -0.00223017,
+                -0.00264033,
+                -0.00201168,
+                -0.00262622,
+                -0.00209184,
+                -0.00253433,
+                -0.00205081,
+            ]
```

Comment:
Could we use the decimals upto 4 precision points? 

You can print it like so:

```python
audio_slice = audio_slice.flatten().tolist()
print(", ".join([str(round(x, 4)) for x in audio_slice]))
```

Reply:
Addressed.